workers/unity/Packages/com.improbable.gdk.movement/Behaviours/ProxyMovementDriver.cs

```diff
@@ -87,5 +86,11 @@ protected override void Update()
                 hasRotationLeft = false;
             }
         }
+
+        public bool GetGrounded()
```

Comment:
Is there a way we can just call CheckGrounded after each movement change is applied, and no need for this method as we can just access IsGrounded? (To avoid doing the check every frame on things that aren't even moving)

Reply:
Yeah, that can be done, but may involve a number of unneeded calculations, if ground checking isn't needed, or multiple `move`s are called in a frame. But I agree, it is neater

Reply:
Is the grounded state currently being called every frame by the animator? If so, the change shoudl lead to less calculations, if anything.